yb-voyager/cmd/assessMigrationCommand.go

```diff
@@ -947,7 +947,7 @@ func areMinVersionsFixedInEqual(m1 map[string]*ybversion.YBVersion, m2 map[strin
 	return true
 }
 
-func getUnsupportedFeaturesFromSchemaAnalysisReport(featureName string, issueReason string, schemaAnalysisReport utils.SchemaReport, displayDDLInHTML bool, description string) UnsupportedFeature {
+func getUnsupportedFeaturesFromSchemaAnalysisReport(featureName string, issueReason string, schemaAnalysisReport utils.SchemaReport, displayDDLInHTML bool, description string) *UnsupportedFeature {
```

Comment:
Instead of changing it to pointer, how about we keep the return type same.
And define EMPTY_UNSUPPORTED_FEATURE and return that, and later filter that out of the final array.

```
var EMPTY_UNSUPPORTED_FEATURE = UnsupportedFeature{}
```

Reply:
Changed the way to make pointer changes, but just filter at the end if there are objects